app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PPOCardView.kt

```diff
@@ -351,6 +388,102 @@ fun CreatorNameSendMessageView(
     KSDividerLineGrey()
 }
 
+@Composable
```

Comment:
This is the main block of code containing the new UI. The rest of the changes after this block are mostly minor cleanup from the parent PR https://github.com/kickstarter/android-oss/pull/2049